src/module.ts

```diff
@@ -3418,16 +3418,32 @@ export class SwitchBuilder {
     this.condition = condition;
   }
 
+  /** Links a case to the specified branch, replace old case if it is linked. */
+  addOrReplaceCase(value: i32, code: ExpressionRef[]): void {
+    let valueIndex = this.values.indexOf(value);
```

Comment:
```suggestion
    const valueIndex = this.values.indexOf(value);
```

Reply:
AS for the most part uses `let` instead of `const`. I like `const` as well; should we be using that when possible from now on?

Reply:
If I'm not mistaken, it was meaningful in the past, but I don't think it does now. It's just that in one place there is already `const` so for consistency I suggested replacing it here.

Reply:
Ah, got it